### DISCUSSION TITLE
Can I just copy the SD card contents to another device? (YI Pro 2K Home)

### DISCUSSION THREAD

**Discussion by naddel91** [roleoroleo/yi-hack-Allwinner-v2#941] [General]
Hi,

in case I wanted to just copy the config (including gdrive addon) to another device (same model), can I simply do this and have it work the same way (including gdrive uploads)?

**Comment by roleoroleo** [comment]
About cam settings, you can save a backup using the webgui.
About gdrive you must copy gdrive folder manually.

**Comment by naddel91** [comment]
what if I copy everything on my SD card manually to another device. same as just transfering the SD card to another device. will this work?

**Reply by roleoroleo** [reply]
Yes, if the other device is the same model.
y21ga, h52ga, etc...
